Readme.md

```diff
@@ -194,6 +198,21 @@ fn("/invalid"); //=> false
 fn("/user/caf%C3%A9"); //=> { path: '/user/caf%C3%A9', index: 0, params: { id: 'café' } }
 ```
 
+The `match` function can be used to custom match named parameters (as of Node 10.0.0). For example, this can be used to whitelist a small number of valid paths:
```

Comment:
I personally don't know what you mean by "(as of Node 10.0.0)" here. 
I ran the example code under Node v9.11.2 and got the same output. Can you elaborate for me?

Reply:
Hmm, I must've been confusing that limitation with that of another function:
https://github.com/pillarjs/path-to-regexp/pull/225

Will make the change.